services/QuillLMS/Gemfile

```diff
@@ -142,28 +139,36 @@ gem 'rack-heartbeat'
 # INTEGRATIONS
 gem 'clever-ruby', '~> 2.0.1'
 
+# Memory profiling
+gem 'puma_worker_killer', '~> 0.1.0'
+
+# temp for migrations
+gem 'paperclip'
+
+# TBD
+gem 'rb-readline'
+gem 'awesome_print'
+
```

Comment:
I like the consolidation you did here for items we should do later analysis on regarding whether we use these or not.  It may be worth going ahead and creating a Task card in Notion so that we remember to come back and do this.

Reply:
I'm planning on doing an audit after the 6.1 upgrade.